omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml

```diff
@@ -0,0 +1,1531 @@
+                    var adherence = jq(this).find("input[type='radio']:checked").val();
+                    if(adherence == 90157 || adherence == 90158) {
+                        jq(".EnableDisableAdherenceReason").find("input[type$='checkbox']").attr('disabled', false);
+                        //jq(".EnableDisableAdherenceReason").removeClass("html-form-entry-disabled-field");
+                    } else {
+                        jq(".EnableDisableAdherenceReason").find("input[type$='checkbox']").attr('disabled', true);
+                        //jq(".EnableDisableAdherenceReason").addClass("html-form-entry-disabled-field");
+                    }
+                });
+
+                /*enable disable Hepatitis B status*/
+
+                jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', true);
+
+                jq(".HepBStatus").change(function () {
+                    var hepStatus = jq(this).find("input[type='radio']:checked").val();
+                    if(hepStatus == 159971) {
+                        jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', false);
+                    } else {
+                        jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', true);
+                    }
+
+                });
+
+                jq("#bmi").find("input[type$='text']").attr('disabled', true);
+
+                jq(".enableDisable").each(function () {
+                    var group = jq(this);
+
+                    function disableFn() {
+                        group.children("#disabled").fadeTo(250, 0.25);
+                        /*  group.children("#disabled").find(":checkbox").attr("checked", false); uncheck */
+                        group.children("#disabled").find("input").attr("disabled", true);
+                        /* disable */
+                        group.children("#disabled").find("select").attr("disabled", true);
+                    }
+
+                    function enableFn() {
+                        group.children("#disabled").fadeTo(250, 1);
+                        group.children("#disabled").find("input").attr("disabled", false);
+                        group.children("#disabled").find("select").attr("disabled", false);
+
+                    }
+
+                    if (jq(this).children("#trigger").find(":checkbox:first").attr("checked") != "checked") {
+                        disableFn();
+                    }
+
+                    jq(this).children("#trigger").find(":checkbox:first").change(function () {
+                        //var checked = jq(this).attr("checked");
+                        if (jq(this).prop('checked')) {
+                        enableFn();
+                        } else {
+                        disableFn();
+                        }
+                    });
+                });
+
+                function stabilityCheck() {
```

Comment:
@semujju rename to DSDMStability check and add comments to describe what it does, talk to @slubwama

Reply:
Done